src/EnergyPlus/HeatBalanceSurfaceManager.cc

```diff
@@ -2627,14 +2627,20 @@ void InitSolarHeatGains(EnergyPlusData &state)
                                 state.dataSurface->SurfReflFacBmToDiffSolGnd)); // For linear indexing
         Real64 GndReflSolarRad = 0.0;
         Real64 GndSolarRadInc = max(state.dataEnvrn->BeamSolarRad * state.dataEnvrn->SOLCOS(3) + state.dataEnvrn->DifSolarRad, 0.0);
+
+        for (int SurfNum = 1; SurfNum <= state.dataSurface->TotSurfaces; ++SurfNum) {
+            state.dataSurface->Surface(SurfNum).IncSolMultiplier = GetSurfIncidentSolarMultiplier(state, SurfNum);
+        }
```

Comment:
I know this has been merged, but I'm noticing this as I resolve conflicts over in the space heat balance branch. There should be a global flag for `AnySolarMultipliers` that can be used to skip this entire loop if they are not being used (which is the common case).
Also, I would assume these apply only to exterior windows? If so, the loop can be `for(int SurfNum : state.dataSurface->AllExtSolWindowSurfaceList)`.

Reply:
Make sense. I'll add in a global flag to skip the whole loop, and also restrict the looping to within state.dataSurface->AllExtSolWindowSurfaceList